core-blocks/more/edit.js

```diff
@@ -38,7 +38,9 @@ export default class MoreEdit extends Component {
 		const toggleNoTeaser = () => setAttributes( { noTeaser: ! noTeaser } );
 		const { defaultText } = this.state;
 		const value = customText !== undefined ? customText : defaultText;
-		const inputLength = value.length + 1;
+
+		// Set the input width to 3em per character, plus 1em to allow for empty strings.
```

Comment:
`em` is a vertical measure, isn't it? Maybe more accurate-ish to use `ch` ?

https://developer.mozilla.org/en-US/docs/Web/CSS/length#Font-relative_lengths

Reply:
@jasmussen, @kjellr or @mapk any thoughts on this comment?

Reply:
I've never used the `ch` measurement before, but after seeing its measurement is based on the `0` while the `em` is based on font-size, I'm up in the air on this.